README.md

```diff
@@ -6,38 +6,26 @@ This library can receive and parse data from JIRA webhook, convert it into easil
 It comes loaded with a few [default listeners](#default-listeners) for the most common scenarios, but you can [extend it](#customizing) with your own custom logic too.
 
 
-# Installation
-To install this library clone the code from GitHub and run `composer install` in the root folder.
+# Installation and configuration
+To install this library simply pull it in through composer
+```        
+    "require": {
+        "kommuna/vicky": "dev-master",
+    },
+```   
+and run `composer install`.
 
 #### 1. Config files  
 Copy `bot/config.example.php` to `/etc/SlackBot/config.php` and `client/config.example.php` to `/etc/vicky/config.php` and follow the instructions in them to set your values.
 
-Get your bot token [here](https://my.slack.com/services/new/bot).
+You can get your slack bot token [here](https://my.slack.com/services/new/bot).
```

Comment:
😸  Personally.. I actually like the human-to-human approach, but you're right, the spec should be technical. I changed this to "The slack bot token can be obtained here".
There are other instances of "you" though...  For example: 
>"If you need to handle specific JIRA situations in your own, specific ways, you can register your own listeners." 
How do you think I should change this so it sounds more technical? 
@l80